jekyll/_cci2/contexts.md

```diff
@@ -48,8 +48,51 @@ workflows:
 
 To rerun a job and use the context, it **must** be rerun from the Workflows page of the CircleCI application.
 
+## Securing a Context
+
+CircleCI enables you to secure secrets at run time by adding security groups to contexts. Only Org Admins may add *security groups* to a new or existing context. Security groups are definied as LDAP groups or GitHub teams. After the security group is added to a context, only members of that security group may access or use the environment variables of the secure context. 
+
+The default security group is `organization` and enables any member of the org to use the context.
+
+## Runnning Workflows with a Secure Context
+
+To invoke a workflow that uses a secure context, a user must be a member of one of the security groups for the context or the workflow will fail with the status of `Unauthorized`. If you add a context to your workflow and you are **not** a member of any of the context groups, the job will fail as `Unauthorized`.
+
+**Note:** Bitbucket repositories do **not** provide an API that allows CircleCI contexts to be secured, only GitHub hosted projects or projects that use GitHub Enterprise with CircleCI include the ability to limit contexts with security groups.
+
+### Limit a Context to a Security Group or Groups
+
+You must be an Org Admin to complete the following task.
```

Comment:
That is not really true. Anyone can create a context but only org admins can secure them.

Reply:
Right, the create a context task is earlier in the document, this task is specific to securing the context